packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -231,6 +245,10 @@ export function createLiveblocksContext<
       throw fetchInboxNotifications();
     }
 
+    if (query.error !== undefined) {
+      throw query.error;
+    }
```

Comment:
We had missed this somehow. Figured out when I was writing a test to verify error boundary was triggered.